src/clj/game/cards/assets.clj

```diff
@@ -185,6 +185,22 @@
                                       card nil)))}}
       :abilities [ability]})
 
+   "API-S Keeper Isobel"
+    (letfn [(counters-available? [state] (some #(pos? (get-counters % :advancement)) (all-installed state :corp)))]
+      {:flags {:corp-phase-12 (req (counters-available? state))}
+       :abilities [{:req (req (and (:corp-phase-12 @state)
+                                   (counters-available? state)))
+                    :once :per-turn
+                    :label "Remove an advancement token from an installed card and gain 3 [Credits]"
```

Comment:
I would prefer this shorter, and with an indication that it's a start-of-turn ability. "Remove an advancement token (start of turn)" would be my suggestion.